crates/astria-sequencer/src/app.rs

```diff
@@ -733,6 +734,28 @@ impl App {
     }
 }
 
+/// Display wrapper for `HashMap<RollupId, Vec<Deposit>>`.
+struct DisplayDeposits<'a>(&'a HashMap<RollupId, Vec<Deposit>>);
+
+impl<'a> std::fmt::Display for DisplayDeposits<'a> {
```

Comment:
When writing a `Display` impl for these complex types, please ensure that they are done using a json formatting. That makes analyzing them easier (once grafana etc is working).

I suggest deriving `serde::Serialize` with your preferred way of formatting the map. Alternatively, impl your own `serde::Serialize` for your reporting newtype wrapper, and then use `astria_telemetry::display::json` to emit the JSON in tracing.